Detectors/TPC/reconstruction/src/RawReader.cxx

```diff
@@ -141,12 +153,30 @@ bool RawReader::loadEvent(int64_t event) {
   mData.clear();
 
   auto ev = mEvents.find(event);
-  mLastEvent = event;
 
   if (ev == mEvents.end()) return false;
+  mLastEvent = event;
   
   const Mapper& mapper = Mapper::instance();
   for (auto &eventInfo : *(ev->second)) {
```

Comment:
with "const auto &eventInfo" somehow "eventInfo.header.timeStamp()" doesn't compile anymore 

/home/tpc/alice/sw/SOURCES/O2/dev/0/Detectors/TPC/reconstruction/src/RawReader.cxx:173:111: error: passing 'const o2::TPC::RawReader::Header' as 'this' argument discards qualifiers [-fpermissive]
         << "0x" << std::hex << std::setfill('0') << std::right << std::setw(16) << eventInfo.header.timeStamp() << FairLogger::endl;